server/app.mjs

```diff
@@ -356,7 +366,7 @@ const initGPTScriptConfig = async (gptscript) => {
 
     const configPath = gptscriptConfigPath();
 
-    fs.readFile(configPath, 'utf8', (err, data) => {
+    fs.readFile(configPath, 'utf8', async (err, data) => {
```

Comment:
We want `initGPTScriptConfig` to block server startup until the file is created. Won't switching this to async mean that `initGPTScriptConfig` can return before the file is created?

Reply:
Idk but I'll remove it and handle everything with `.then`.